### PR TITLE
Ab#103480 view request board

### PR DESCRIPTION
**Pull request by TC-Cory**

### CODE REVIEW

File: src/CSF.SRDashboard.Client/Components/Tables/WorkloadRequest/WorkloadRequestTableItemPriorityComponent.razor.cs [hunk 1]
```diff
@@ -15,20 +15,20 @@ public partial class WorkloadRequestTableItemPriorityComponent
...
+       /* public bool HasWarning
         {
             get
             {
-                return !string.IsNullOrEmpty(this.Priority) && this.Priority.Equals("missing info", StringComparison.InvariantCultureIgnoreCase);
+                return this.Priority.Equals("Urgent", StringComparison.InvariantCultureIgnoreCase);
             }
         }
-
+*/
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Components/Tables/WorkloadRequest/WorkloadRequestTableItemPriorityComponent.razor.cs:25]: remove commented code.

File: src/CSF.SRDashboard.Client/Components/Tables/WorkloadRequest/WorkloadRequestTableItemPriorityComponent.razor.cs [hunk 2]
```diff
@@ -15,20 +15,20 @@ public partial class WorkloadRequestTableItemPriorityComponent
...
         public string GetTextCssClass()
         {
             if (this.HasError)
                 return "text-danger";
-            if (this.HasWarning)
-                return "text-warning";
+           /* if (this.HasWarning)
+                return "text-warning";*/
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Components/Tables/WorkloadRequest/WorkloadRequestTableItemPriorityComponent.razor.cs:31]: remove commented code

File: src/CSF.SRDashboard.Client/Pages/WorkloadRequests.razor.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        public List<WorkloadRequestTableItem> WorkloadData { get; set; }
+        List<WorkItemDTO> workItems = new List<WorkItemDTO>();
+        [Inject]
+
+        public IWorkLoadManagementService WorkLoadService { get; set; }
+
+        protected override void OnInitialized()
+        {
+            base.OnInitialized();
+            //
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Pages/WorkloadRequests.razor.cs:23]:
```suggestion
```

File: src/CSF.SRDashboard.Client/wwwroot/css/site.css [hunk 4]
```diff
@@ -261,7 +261,41 @@ span.csf-span {
...
+}
+.badge-inprogress {
+    color: black;
+    font-weight: 100;
+    background-color: #D7BDE5CC;
+    border: 1px solid #C4A8D9;
+}
+.workloadtableheader {
+    font-weight: 700;
+    background-color: #DEE2E6;
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/wwwroot/css/site.css:296]: We may want to use the bootstrap stylings in the future, but for now, this is acceptable.

**Review by AlexandreSanscartier — CHANGES_REQUESTED** [review]
Minor changes other than that everything looks good.

File: src/CSF.SRDashboard.Client/wwwroot/css/site.css (hunk 4 above)
Comment (TC-Cory) [reply src/CSF.SRDashboard.Client/wwwroot/css/site.css:296]: This is following the design from UX, I can change is back if needed.

**Review by AlexandreSanscartier — APPROVED** [review]

**Pull request merged**